runtime/src/lib.rs

```diff
@@ -645,7 +645,7 @@ construct_runtime! {
 		Identity: pallet_identity::{Module, Call, Storage, Event<T>},
 
         // Crust modules
-        Swork: swork::{Module, Call, Storage, Event<T>, Config<T>},
+        Swork: swork::{Module, Call, Storage, Event<T>, Config},
```

Comment:
just curious about the difference between `Config` and `Config<T>`

Reply:
If we have storage with **generic type** to initialize(config), we need this `<T>`